rerun_cpp/src/rerun/recording_stream.cpp

```diff
@@ -1,12 +1,11 @@
 #include "recording_stream.hpp"
 
-#include "data_cell.hpp"
-#include "rerun.h"
+#include <rerun.h>
```

Comment:
My memory is that including a project-internal header via `<>` can occasionally lead to some painful edge-cases if you have a conflicting system-wide rerun installation. I would be inclined to use "../rerun.h" here or if we want to avoid upwards-relative-paths, to move rerun.h down a level, potentially making `#include <rerun/rerun.h>` the standard public include directive.

Reply:
I agree and share that experience  *but* there's a subtly here! This is **rerun.h** not rerun.hpp, i.e the C library header. The C library is separate and set a system include for this library and at this point we don't know where it lives.